scripts/pull_github_pr.sh

```diff
@@ -36,6 +48,52 @@ curl() {
     command curl "${opts[@]}" "$@"
 }
 
+set_jenkins_job() {
+  branch=$(git rev-parse --abbrev-ref HEAD)
+  product=$(cat SCYLLA-VERSION-GEN | grep ^PRODUCT | awk -F'=' '{print $2}')
```

Comment:
This is not critical, but I want to point out that I use an old version of our pull_github_pr.sh not just in Scylla, but also in Seastar and even OSv (!) - it's a damn useful script (thanks @avikivity) to replace the nasty "merge" button on github. And this 100%-Scylla-specific code is breaking this usefulness. I would have split the script into two parts, but even if not, it would be nice to make *this* part more forgiving - i.e., if SCYLLA-VERSION-GEN you could just ignore all this Jenkins stuff, instead of failing in weird ways.

Oh, and instead of using cat grep **and** awk, you could do all of this with one command:
```
awk -F'=' '/^PRODUCT/ {print $2}' <SCYLLA-VERSION-GEN
```

Reply:
changed